src/database/videos.js

```diff
@@ -1,12 +1,18 @@
 import { useAuthStore } from '../hooks/useAuthStore';
 
-const videos = async () => {
+export const videos = async () => {
   const { videosLearningPath } = useAuthStore;
   const videosrender = videosLearningPath.map((videos) => {
```

Comment:
Could you please replace `videosrender` by `videosRender` please?

```suggestion
  const videosRender = videosLearningPath.map((videos) => {
```